docs(readme): Apdex integration (#11655)
GoogleChrome/lighthouse: readme.md

* **[Foo](https://www.foo.software/lighthouse)** - Lighthouse-as-a-service offering free and premium plans. Provides monitoring and historical reporting of Lighthouse audits with CircleCI, GitHub, and other integrations. Features include Slack notifications, PR comment reporting and more.

> * **[Apdex](https://apdex.co)** - Apdex is a website performance service. The main features are historical Lighthouse reports visualizations, mobile/desktop options, alerts, uptime monitoring, and more. There are flexible plans and a 30-day free trial.

Reviewer: ```suggestion
* **[Apdex](https://apdex.co)** - Apdex is a website performance service. The main features are historical Lighthouse report visualizations, mobile/desktop options, alerts, uptime monitoring, and more. There are flexible paid plans and a 30-day free trial.
```

> * **[Apdex](https://apdex.co/performance)** - Apdex is website performance service. Provides historical Lighthouse reports visializations, global regions, mobile/desktop option, custom headers and alerts. It's also all-in-one solution, providing website uptime monitoring and more. Apdex is paid product with free 30-day trial and flexible plans.

Reviewer: ```suggestion
* **[Apdex](https://apdex.co/performance)** - Apdex is a website performance monitoring service. Provides historical Lighthouse report visualizations, global region options, mobile+desktop support, custom headers, alerts, and uptime monitoring. Apdex is paid product with free 30-day trial and flexible plans.
```

just some consolidation and minor grammar fixes for you to consider :)
